src/components/SubHeader/Restaurant/Header/Header.js

```diff
@@ -0,0 +1,31 @@
+import PropTypes from "prop-types";
+import "./Header.css";
+
+const Header = (props) => {
+  const { name, category, street, locality, children } = props;
+
+  return (
+    <div className="restaurant-about flex-column justify-content-center">
+      <h3>{name}</h3>
+      <p>
+        <span>Closes Soon</span>
+        &nbsp;
+        <span>{category}</span>
+      </p>
+      <p>
+        {street}, {locality}
+      </p>
+      <div className="flex align-items-center">{children}</div>
```

Comment:
use renderProp to render this, take a function which returns you react elements which you will populate here